docs/mobile-apps/features/network-capture.md

```diff
@@ -7,92 +7,182 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
+<img src={useBaseUrl('img/mobile-apps/network-capture-request-details.png')} alt="Inspect network logs" width="700"/>
+
+#### Stats Row
+The Stats row in the footer provides details on the number of requests, transferred data size, resource sizes, and time metrics such as
+Page Load, DOMContentLoaded, and Finished time.
+<img src={useBaseUrl('img/mobile-apps/network-capture-stats-row.png')} alt="Inspect network logs" width="700"/>
+
+### Limitations
+:::note Limitations
+
+- iOS network capture is supported on iOS/iPadOS 14.0 and above.
+- Android network capture is supported on Android 10 and above.
+- Only HTTP/HTTPS network traffic is captured.
 
-<img src={useBaseUrl('img/mobile-apps/network-capture-4.png')} alt="Inspect network logs" width="650"/>
+:::
 
 ## Accessing HAR Files (Live and Automated Testing)
 
 To download HAR files from Sauce Labs:
 
 1. Navigate to **Test Results**.
 2. Select the applicable test.
-3. Click the **Network** tab in the bottom to preview network calls.
-4. Click the **Logs** tab on the right panel and select **network.har** from the dropdown menu to download the report.
+3. Click the **Network** tab on the right panel to preview all captured network requests.
 
 You can also download a HAR file programmatically using the following API request:
 
-```curl
-curl --compressed \
--O https://{SAUCE_USERNAME}:{SAUCE_ACCESS_KEY}@{DATA_CENTER}.saucelabs.com/v1/rdc/jobs/{JOB_ID}/network.har
+<Tabs
+groupId="dc-url"
+defaultValue="us"
+values={[
+{label: 'United States', value: 'us'},
+{label: 'Europe', value: 'eu'},
+]}>
+
+<TabItem value="us">
+
+```jsx title="Sample Request"
+curl -u "$SAUCE_USERNAME:$SAUCE_ACCESS_KEY" --location \
+--request GET 'https://api.us-west-1.saucelabs.com/v1/rdc/jobs/{JOB_ID}/network.har'
 ```
 
-### Viewing HAR Files
+</TabItem>
 
-You'll need a HAR viewer to read HAR files. Sauce Labs provides a [React Network Viewer](https://opensource.saucelabs.com/network-viewer/) HAR viewer for your convenience. For step-by-step usage instructions, see our [Visualize HAR Files with the New React Network Viewer](https://opensource.saucelabs.com/blog/react_network_viewer/) blog article.
+<TabItem value="eu">
 
-Alternatively, some other commonly used HAR viewers include:
+```jsx title="Sample Request"
+curl -u "$SAUCE_USERNAME:$SAUCE_ACCESS_KEY" --location \
+--request GET 'https://api.eu-central-1.saucelabs.com/v1/rdc/jobs/{JOB_ID}/network.har'
+```
 
-- Chrome devtools builtin [HAR imports](https://developer.chrome.com/blog/new-in-devtools-62/#har-imports)
-- Google Admin Toolbox [HAR Analyzer](https://toolbox.googleapps.com/apps/har_analyzer/)
-- Software is Hard blog [HTTP Archive Viewer](http://www.softwareishard.com/har/viewer/)
-- Firefox network monitoring builtin [HAR imports](https://firefox-source-docs.mozilla.org/devtools-user/network_monitor/toolbar/index.html)
+</TabItem>
+</Tabs>
 
-## HAR Files
+## Instrumented Network Traffic Capture
```

Comment:
Do we need to clarify again that this is only for Android?

Reply:
yes